### ISSUE TITLE
No module named enum

### ISSUE THREAD

**Issue by ncook-hxgn** [johnthagen/cppcheck-junit#13]
Hi,

Just installed this module
`E:\repos\TestProject>pip install cppcheck_junit`

.. then updated pip because it told me to:
`E:\repos\TestProject>python -m pip install --upgrade pip`

Then I tried to convert a file as instructed in README.md, got this:
```cmd
E:\repos\TestProject>cppcheck_junit cppcheck-result.xml cppcheck-junit.xml
Traceback (most recent call last):
  File "c:\python27amd64\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\python27amd64\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Python27amd64\Scripts\cppcheck_junit.exe\__main__.py", line 5, in <module>
  File "c:\python27amd64\lib\site-packages\cppcheck_junit.py", line 16, in <module>
    from exitstatus import ExitStatus
  File "c:\python27amd64\lib\site-packages\exitstatus.py", line 3, in <module>
    import enum
ImportError: No module named enum

E:\repos\TestProject>
```

I'm using Windows Terminal on Windows 10 Pro, probably Python 2.7.16 (blame googles various build toolings).

**Comment by johnthagen** [comment]
Hello, this project only supports Python 3.6+. The `enum` module does not exist in Python 2.7 standard library, so you'll need to upgrade to a supported Python version.

For Windows, these can be found here: https://www.python.org/downloads/

**Comment by ncook-hxgn** [comment]
Hi @johnthagen 

Sorry it took ages to get back to you!

I can confirm: I have both Python 2.7 and Python 3.9, but having built GoogleTest recently, I'd modified my path to find python 2.7 first.  🤹‍♀️

I've now corrected my path, cursed the irony of Google build tools using Python 2 when they invented Go , and will report back when I have confirmed what I am sure is going to be a successful run of the tool.

Many thanks for pointing out the obvious to me - I needed it!
